devicetypes/sinope-technologies/va4200wz-va4200zb-sinope-valve.src/va4200wz-va4200zb-sinope-valve.groovy

```diff
@@ -286,6 +281,32 @@ private int get_LOG_TRACE() {
 	return 5
 }
 
+private def convertVoltToPercent(value) {
+	def levelValue;
+	def levelsTable = [0, 20000, 40000, 60000, 80000, 100000];
+	def anglesTable = [30, 55, 56, 57, 58.5, 60];
+
+	if (value > anglesTable[anglesTable.size - 1]) { // if the value of the angle is greater than the maximum
+	  value = anglesTable[anglesTable.size - 1]; // use the maximum value instead
+	}
+
+	def index = 1;
+    while ( value > anglesTable[index]) { index++ }
```

Comment:
Fix indentation.

Reply:
For some reason I can't seem to be able to correct my PR. Here is the new PR corrected : https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/73935

Thanks!

Reply:
> For some reason I can't seem to be able to push my correction on the PR. Here is the new PR corrected : #73935
> 
> Thanks!

Thank you @vbeauregard-Sinope , we will review https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/73935. Could you please close https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/72607 ? Many thanks